docs/word/word-add-ins-events.md

```diff
@@ -0,0 +1,199 @@
+| `onCommentDeleted` | Occurs when comments are deleted.<br><br>Event data object:<br>[CommentEventArgs](/javascript/api/word/word.commenteventargs) | <ul><li>[**Body**](/javascript/api/word/word.body?view=word-js-preview&preserve-view=true#word-word-body-oncommentdeleted-member)</li><li>[**Paragraph**](/javascript/api/word/word.paragraph#word-word-paragraph-oncommentdeleted-member)</li></ul> |
+| `onCommentDeselected` | Occurs when a comment is deselected.<br><br>Event data object:<br>[CommentEventArgs](/javascript/api/word/word.commenteventargs) | <ul><li>[**Body**](/javascript/api/word/word.body?view=word-js-preview&preserve-view=true#word-word-body-oncommentdeselected-member)</li><li>[**ContentControl**](/javascript/api/word/word.contentcontrol?view=word-js-preview&preserve-view=true#word-word-contentcontrol-oncommentdeselected-member)</li><li>[**Paragraph**](/javascript/api/word/word.paragraph#word-word-paragraph-oncommentdeselected-member)</li><li>[**Range**](/javascript/api/word/word.range#word-word-range-oncommentdeselected-member)</li></ul> |
+| `onCommentSelected` | Occurs when a comment is selected.<br><br>Event data object:<br>[CommentEventArgs](/javascript/api/word/word.commenteventargs) | <ul><li>[**Body**](/javascript/api/word/word.body?view=word-js-preview&preserve-view=true#word-word-body-oncommentselected-member)</li><li>[**ContentControl**](/javascript/api/word/word.contentcontrol?view=word-js-preview&preserve-view=true#word-word-contentcontrol-oncommentselected-member)</li><li>[**Paragraph**](/javascript/api/word/word.paragraph#word-word-paragraph-oncommentselected-member)</li><li>[**Range**](/javascript/api/word/word.range#word-word-range-oncommentselected-member)</li></ul> |
+
+### Event triggers
+
+Events within a Word document can be triggered by:
+
+- User interaction via the Word user interface (UI) that changes the document.
+- Office Add-in (JavaScript) code that changes the document.
+- VBA add-in (macro) code that changes the document.
+
+Any change that complies with default behavior of Word will trigger the corresponding events in a document.
+
+### Lifecycle of an event handler
+
+An event handler is created when an add-in registers the event handler. It's destroyed when the add-in deregisters the event handler or when the add-in is refreshed, reloaded, or closed. Event handlers don't persist as part of the Word file, or across sessions with Word on the web.
+
+> [!CAUTION]
+> When an object to which events are registered is deleted (e.g., a content control with an `onDataChanged` event registered), the event handler no longer triggers but remains in memory until the add-in or Word session refreshes or closes.
```

Comment:
Is this a cautionary statement because of memory issues?

Reply:
What should a reader do with this information? I expect this is pretty esoteric/non-actionable for them? My inclination would be to skip this.